integration/tests/fargate/fargate_test.go

```diff
@@ -148,39 +148,6 @@ var _ = Describe("(Integration) Fargate", func() {
 			deleteCluster(ft.clusterName)
 		})
 	})
-
-	Context("Creating a cluster with --fargate and --managed", func() {
-		ft := &fargateTest{}
-
-		BeforeEach(func() {
-			setup(ft, "--fargate", "--managed")
-		})
-
-		It("should support Fargate", func() {
-			testDefaultFargateProfile(ft.clusterName, ft.kubeTest)
-			testCreateFargateProfile(ft.clusterName, ft.kubeTest)
-		})
-
-		AfterEach(func() {
-			deleteCluster(ft.clusterName)
-		})
-	})
```

Comment:
I didn't really get the point of this tbh why does testing fargate when a managed nodegroup exists need its own test? Its also tested in unowned suite